### ISSUE TITLE
 Lable::createWithSystemFont with chinese show as  block  on windows

### ISSUE THREAD

**Issue by ychua842655** [cocos2d/cocos2d-x#17377]
- cocos2d-x version: 3.13.1
- devices test on: windows 10
- developing environments
   - NDK version:
   - Xcode version:
   - VS version:  vs2015
   - browser type and version:

Lable::createWithSystemFont with chinese show as block on windows ,
Lable::createWithTTF is work fine;
Picture show below is a screenshot of my App run on windows
![c805c81995f41f63783bae5fb2707281](https://cloud.githubusercontent.com/assets/8856871/23024362/eca12a30-f494-11e6-90fc-96eea62730fd.jpg)

@minggo 
  The error description is wrong,Label show Chinese only worked with owner .ttf file whatever APi createWithSystemFont or createWithTTF, and i found that word wrap for chinese is not work

**Comment by minggo** [comment]
did you write the characters in source code?

**Comment by ychua842655** [comment]
No, I get it from utf8 file.And it work's fine with api createWithTTF

**Comment by minggo** [comment]
Any codes to reproduce it? Did you pass system font name, may be your system doesn't have the font you specified.

**Comment by ychua842655** [comment]
@minggo 
I pass the system font name as "Helvetica",just like as ios/android .
test code is：
     	std::string str = FileUtils::getInstance()->getStringFromFile("testFile.txt");
	auto label = Label::createWithSystemFont(str,"Helvetica",20,Size(getContentSize().width/4,0));
	label->setPosition(getContentSize() / 2);
	this->addChild(label,2);
[testFile.txt](https://github.com/cocos2d/cocos2d-x/files/786390/testFile.txt)
This is my test file with chinese utf8 string

**Comment by minggo** [comment]
I don't have win10, i will test on win7 to take a look.

**Comment by ychua842655** [comment]
Thank you

**Comment by minggo** [comment]
I tested with latest codes on window7, no problem. I will test on v3.13.1 branch.

**Comment by ychua842655** [comment]
Thank you ,Dose results  about v3.13.1 branch out now?

**Comment by minggo** [comment]
windows7 also works on v3.13.1, so the problem may be something related with windows10. Yep, latest version is v3.14.1.

**Comment by ychua842655** [comment]
Thank you very much,I will test it on windows phone 10 and give the feedback to you

**Comment by ychua842655** [comment]
@minggo  If you can, please test it on win10 environment.（will it be a question of access?）

**Comment by summerinsects** [comment]
Hey here!
I reproduced it! The reason why it displayed as square is the ttf your specified does not support Chinese.
Maybe you can download Arial Unicode MS (21.8 MB), and put it into the folder Resources/fonts
